sailfishusermanager.cpp

```diff
@@ -290,11 +291,20 @@ uint SailfishUserManager::addUser(const QString &name)
 
 int SailfishUserManager::removeUserFiles(uint uid)
 {
+    int rv = EXIT_FAILURE;
     QDir dir(USER_ENVIRONMENT_DIR.arg(uid));
     if (dir.removeRecursively())
-        return EXIT_SUCCESS;
-    qCWarning(lcSUM) << "Removing user environment directory failed";
-    return EXIT_FAILURE;
+        rv = EXIT_SUCCESS;
+    else
+        qCWarning(lcSUM) << "Removing user environment directory failed";
+
+    // Execute user removal scripts
+    QDir removal(USER_REMOVE_SCRIPT_DIR);
+    foreach (const QString &entry, removal.entryList(QDir::Files | QDir::NoDotAndDotDot)) {
```

Comment:
Does this actually need `NoDotAndDotDot` when it has `Files`? Then again it doesn't really hurt.